litellm/integrations/langfuse.py

```diff
@@ -203,8 +205,8 @@ def _log_langfuse_v1(
                 endTime=end_time,
                 model=kwargs["model"],
                 modelParameters=optional_params,
-                input=input,
-                output=output,
+                prompt=input,
+                completion=output,
```

Comment:
also fixed parameter names for v1

see https://langfuse.com/docs/sdk/python#rename-prompt-and-completion-to-input-and-output

and https://github.com/langfuse/langfuse-python/blob/v1.14.0/langfuse/api/resources/commons/types/create_generation_request.py#L23-L24